sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBase.g4

```diff
@@ -262,7 +262,7 @@ describeFuncName
     ;
 
 describeColName
-    : identifier ('.' (identifier | STRING))*
+    : identifier
```

Comment:
We should keep it the same, but just issue an exception when users specify nested columns. Also add a test case to verify it